### PR TITLE
fix(react-styles): updated class names to include versions

### LINKED ISSUE #9150: Bug - react-styles - class names need to be updated
**Describe the problem**
A clear and concise description of the problem. Which components are affected?
The component class definitions in react-styles (https://github.com/patternfly/patternfly-react/tree/v5/packages/react-styles/src/css/components) need to be updated to use the v5 prefix.

in addition this [comment](https://github.com/patternfly/patternfly-react/pull/9139#issuecomment-1553442153) from @mcoker should be addressed

> re: the react-styles updates @tlabaj mentioned, I'm not sure where the names for the color vars are coming from - https://github.com/patternfly/patternfly-react/blob/v5/packages/react-styles/src/css/components/Table/inline-edit.css

> We don't have a --global--[color]--number CSS var in core, though it looks like that is how the vars were defined before we added the v5 prefix, too - https://github.com/patternfly/patternfly-react/commit/3879fc757aefeed80b0694365aa20d5ca856a797#diff-d99fb7d33fdb9c29c555e2926969439a2857b3ec7e95c13155c6c21bb3dce1e2
> 
> I can't find an editable table example using composable table, but I see the legacy table example - https://www.patternfly.org/v4/components/table/react-legacy#editable-rows. It doesn't look like any of the classnames in https://github.com/patternfly/patternfly-react/blob/v5/packages/react-styles/src/css/components/Table/inline-edit.css are being applied in edit mode.
> 
> If those colors still need to be applied, --pf-v5-global--[color]--[number] should be --pf-v5-global--palette--[color]-[number] - and the color names should be lower-case.

### PR DESCRIPTION
**Pull request by wise-king-sullyman**
**What**: Closes #9150

**Additional issues**:

### CODE REVIEW

File: packages/react-styles/src/css/components/Table/inline-edit.css [hunk 1]
```diff
@@ -1,60 +1,60 @@
 :root {
-  --pf-v5-global--Blue--50: #def3ff;
-  --pf-v5-global--Blue--200: #7dc3e8;
+  --pf-v5-global--palette--blue--50: #def3ff;
+  --pf-v5-global--palette--blue--200: #7dc3e8;
```
Comment (mcoker) [inline packages/react-styles/src/css/components/Table/inline-edit.css:3]:
```suggestion
  --pf-v5-global--palette--blue-50: #def3ff;
  --pf-v5-global--palette--blue-200: #7dc3e8;
```

File: packages/react-styles/src/css/components/Table/inline-edit.css [hunk 2]
```diff
@@ -1,60 +1,60 @@
...
 }
 
-tr.pf-c-table__editable-row:hover,
-tr.pf-c-table__editable-row.pf-m-editing {
-  background: var(--pf-v5-global--Blue--50) !important;
+tr.pf-v5-c-table__editable-row:hover,
+tr.pf-v5-c-table__editable-row.pf-m-editing {
+  background: var(--pf-v5-global--palette--blue--50) !important;
```
Comment (mcoker) [inline packages/react-styles/src/css/components/Table/inline-edit.css:8]:
```suggestion
  background: var(--pf-v5-global--palette--blue-50) !important;
```

File: packages/react-styles/src/css/components/Table/inline-edit.css [hunk 3]
```diff
@@ -1,60 +1,60 @@
...
 }
-tr.pf-c-table__editable-row:hover td,
-tr.pf-c-table__editable-row.pf-m-editing td {
-  border-bottom: 1px solid var(--pf-v5-global--Blue--200) !important;
-  border-top: 1px solid var(--pf-v5-global--Blue--200) !important;
+tr.pf-v5-c-table__editable-row:hover td,
+tr.pf-v5-c-table__editable-row.pf-m-editing td {
+  border-bottom: 1px solid var(--pf-v5-global--palette--blue--200) !important;
+  border-top: 1px solid var(--pf-v5-global--palette--blue--200) !important;
```
Comment (mcoker) [inline packages/react-styles/src/css/components/Table/inline-edit.css:13]:
```suggestion
  border-bottom: 1px solid var(--pf-v5-global--palette--blue-200) !important;
  border-top: 1px solid var(--pf-v5-global--palette--blue-200) !important;
```

File: packages/react-styles/src/css/components/Table/inline-edit.css [hunk 4]
```diff
@@ -1,60 +1,60 @@
...
 }
-tr.pf-c-table__editable-row:hover td:first-child,
-tr.pf-c-table__editable-row.pf-m-editing td:first-child {
-  border-left: 1px solid var(--pf-v5-global--Blue--200) !important;
+tr.pf-v5-c-table__editable-row:hover td:first-child,
+tr.pf-v5-c-table__editable-row.pf-m-editing td:first-child {
+  border-left: 1px solid var(--pf-v5-global--palette--blue--200) !important;
```
Comment (mcoker) [inline packages/react-styles/src/css/components/Table/inline-edit.css:17]:
```suggestion
  border-left: 1px solid var(--pf-v5-global--palette--blue-200) !important;
```

File: packages/react-styles/src/css/components/Table/inline-edit.css [hunk 5]
```diff
@@ -1,60 +1,60 @@
...
 }
-tr.pf-c-table__editable-row:hover td:last-child,
-tr.pf-c-table__editable-row.pf-m-editing td:last-child {
-  border-right: 1px solid var(--pf-v5-global--Blue--200) !important;
+tr.pf-v5-c-table__editable-row:hover td:last-child,
+tr.pf-v5-c-table__editable-row.pf-m-editing td:last-child {
+  border-right: 1px solid var(--pf-v5-global--palette--blue--200) !important;
```
Comment (mcoker) [inline packages/react-styles/src/css/components/Table/inline-edit.css:21]:
```suggestion
  border-right: 1px solid var(--pf-v5-global--palette--blue-200) !important;
```

File: packages/react-styles/src/css/components/Table/inline-edit.css [hunk 6]
```diff
@@ -1,60 +1,60 @@
...
 }
-tr.pf-c-table__editable-row.pf-m-table-editing-first-row {
-  border-top: 3px solid var(--pf-v5-global--Blue--200) !important;
+tr.pf-v5-c-table__editable-row.pf-m-table-editing-first-row {
+  border-top: 3px solid var(--pf-v5-global--palette--blue--200) !important;
```
Comment (mcoker) [inline packages/react-styles/src/css/components/Table/inline-edit.css:24]:
```suggestion
  border-top: 3px solid var(--pf-v5-global--palette--blue-200) !important;
```

File: packages/react-styles/src/css/components/Table/inline-edit.css [hunk 7]
```diff
@@ -1,60 +1,60 @@
...
 }
-tr.pf-c-table__editable-row.pf-m-table-editing-last-row {
-  border-bottom: 3px solid var(--pf-v5-global--Blue--200) !important;
+tr.pf-v5-c-table__editable-row.pf-m-table-editing-last-row {
+  border-bottom: 3px solid var(--pf-v5-global--palette--blue--200) !important;
```
Comment (mcoker) [inline packages/react-styles/src/css/components/Table/inline-edit.css:27]:
```suggestion
  border-bottom: 3px solid var(--pf-v5-global--palette--blue-200) !important;
```

File: packages/react-styles/src/css/components/Table/inline-edit.css [hunk 8]
```diff
@@ -1,60 +1,60 @@
...
-.pf-c-table__inline-edit-buttons {
+.pf-v5-c-table__inline-edit-buttons {
   position: fixed;
   z-index: 1000;
   padding: 4px;
   margin: 0;
-  background: var(--pf-v5-global--Blue--50);
-  border: 1px solid var(--pf-v5-global--Blue--200);
+  background: var(--pf-v5-global--palette--blue--50);
+  border: 1px solid var(--pf-v5-global--palette--blue--200);
```
Comment (mcoker) [inline packages/react-styles/src/css/components/Table/inline-edit.css:44]:
```suggestion
  background: var(--pf-v5-global--palette--blue-50);
  border: 1px solid var(--pf-v5-global--palette--blue-200);
```

**Review by mcoker — CHANGES_REQUESTED** [review]
None of these classes exist in core, so I tried tracing at least one back and looks like this update should work...

The styles for `.pf-v5-c-table__inline-edit-buttons`, as an example, are converted to a js file and that's read in here, and the class is applied to the markup. So updating the source CSS should also update the markup, at least for that class. I'm assuming the `.pf-v5-c-table__editable-row` `pf-v5-c-console` work the same way? I didn't verify that. Also it's worth noting that I found this snippet in the `main` branch - I'm assuming this inline-edit-extension dir has been moved to another repo? It isn't on v5 as far as I can tell.

https://github.com/patternfly/patternfly-react/blob/a0f857c4de39dd415792a8701e7c6ac7fd853024/packages/react-inline-edit-extension/src/components/ConfirmButtons/ConfirmButtons.tsx#L57

However, the color vars do need to be updated. The format is `--pf-v5-global--palette--[color]-[number]` - just need to remove the extra hyphen between the color name and number.

**Review by tlabaj — COMMENTED** [review]
lgtm
I will defer to css for final approval.

**Review by tlabaj — APPROVED** [review]

File: packages/react-styles/src/css/components/Table/inline-edit.css [hunk 9]
```diff
@@ -1,60 +1,60 @@
...
-tr.pf-c-table__editable-row.pf-m-table-editing-last-row {
-  border-bottom: 3px solid var(--pf-v5-global--Blue--200) !important;
+tr.pf-v5-c-table__editable-row.pf-m-table-editing-last-row {
+  border-bottom: 3px solid var(--pf-v5-global--palette--blue-200) !important;
 }
-tr.pf-c-table__editable-row input {
+tr.pf-v5-c-table__editable-row input {
   display: block;
   background: var(--pf-v5-global--BackgroundColor--100);
   border: 1px solid var(--pf-v5-global--BorderColor);
```
Comment (mcoker) [inline packages/react-styles/src/css/components/Table/inline-edit.css:32]:
```suggestion
  border: 1px solid var(--pf-v5-global--BorderColor--100);
```

**Review by mcoker — CHANGES_REQUESTED** [review]
one more small one I missed

**Review by mcoker — APPROVED** [review]
👍

**Review by jpuzz0 — APPROVED** [review]

**Review by gitdallas — APPROVED** [review]

**Pull request merged**